### DISCUSSION TITLE
combat elytra flying

### DISCUSSION THREAD

**Discussion by WhiteProject1** [Zrips/CMI#8723] [Enhancement]
turning off the use of elytra while in combat should be added, if this is added, most people will not prefer other combat plugins like boxpvp server etc.

**Comment by mrfloris** [comment]
it might be possible to utilize the eventcommands on combat start to negate the permission regarding elytra equip, and remove it later again

if combat is only in a specific arena or world, you could use something like this perhaps
```yaml
  # If player will have cmi.worldlimit.elytra.bypass permission node, elytra flight will not be prevented
  # Players joining worlds with disable elytra flight will get their elytra dismounted if possible
  ElytraFlight:
  - run:False
```

**Reply by WhiteProject1** [reply]
hmm let me try
